src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,168 @@
+
+```jsx
+import React from 'react';
+import classNames from 'classnames/bind';
+import { ThemeContext } from 'terra-application/lib/theme';
+import styles from './ThemedComponent.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ExampleComponent = () => {
+  const theme = React.useContext(ThemeContext);
+
+  return (
+    <div className={cx('example-component', theme.className)}>
+      This is an example component.
+    </div>
+  );
+};
+```
+
+## SCSS
+
+Once the theme className has been bound, the component can be appropriately themed. Start by defining the variables for the CSS properties that are expected to change between themes. SCSS variables are defined using [custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*). These values should utilize the SCSS [var()](https://developer.mozilla.org/en-US/docs/Web/CSS/var) functionality. The var() function works by applying a variable if the value has been set. If no value has been set the default value is used.
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// The values in this file represent the default theme. The default theme is applied using the default values in the var() function.
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+To add themes, create new files for each theme and include them in the original SCSS file. Theme are regular SCSS files that define variables to be applied when the theme becomes active. Themes must be locally scoped and must use a supported theme class name. It is recommended to define all available theme variables for each theme. If the component, example component in this case, defines three themable variables, all three variables should be assigned a value in each of the created theme files. This ensures a more robust theme.
+
+`example-component/orion-fusion-theme/ExampleComponent.module.scss`
+
+```scss
+:local {
+  .orion-fusion-theme {
+    --terra-application-example-component-background-color: #f6f8fa;
+    --terra-application-example-component-color: rgb(36, 41, 46);
+    --terra-application-example-component-font-size: 2rem;
+  }
+}
+```
+
+`example-component/clinical-lowlight-theme/ExampleComponent.module.scss`
+
+```scss
+:local {
+  .clinical-lowlight-theme {
+    --terra-application-example-component-background-color: rgb(34, 42, 46);
+    --terra-application-example-component-color: rgb(178, 181, 182);
+    --terra-application-example-component-font-size: 1rem;
+  }
+}
+```
```

Comment:
You could use the codeblock loader to pull in these already defined files.
```jsx
import ClinicalLowLightScss from '.example/clinical-lowlight-theme/ExampleComponent.module.scss?dev-site-codeblock';

<ClinicalLowLightScss />
```

Reply:
Used the style code block loader for each of the examples that demo the full code block:

https://github.com/cerner/terra-application/pull/25/commits/bbbd86df8667fc13a0b7c0ee3ca9997e9365209f